x-pack/legacy/plugins/siem/public/components/timeline/footer/index.tsx

```diff
@@ -161,170 +175,177 @@ export const PagingControl = React.memo<{
+    itemsPerPageOptions.map(item => (
+      <EuiContextMenuItem
+        key={item}
+        icon={itemsPerPage === item ? 'check' : 'empty'}
+        onClick={() => {
+          closePopover();
+          onChangeItemsPerPage(item);
+        }}
+      >
+        {`${item} ${i18n.ROWS}`}
+      </EuiContextMenuItem>
+    ));
+
+  return (
+    <>
+      <FooterContainer
+        data-test-subj="timeline-footer"
+        direction="column"
+        gutterSize="none"
+        height={height}
+        justifyContent="spaceAround"
+      >
+        <FooterFlexGroup
+          alignItems="center"
+          data-test-subj="footer-flex-group"
+          direction="row"
+          gutterSize="none"
+          justifyContent="spaceBetween"
+        >
+          <EuiFlexItem data-test-subj="event-count-container" grow={false}>
+            <EuiFlexGroup
+              alignItems="center"
+              data-test-subj="events-count"
+              direction="row"
+              gutterSize="none"
+            >
+              <EventsCount
+                closePopover={closePopover}
+                isOpen={isPopoverOpen}
+                items={rowItems}
+                itemsCount={itemsCount}
+                onClick={onButtonClick}
+                serverSideEventCount={serverSideEventCount}
+              />
+            </EuiFlexGroup>
+          </EuiFlexItem>
+
+          <EuiFlexItem data-test-subj="paging-control-container" grow={false}>
+            {isLive ? (
+              <EuiText size="s" data-test-subj="is-live-on-message">
+                <b>
+                  {i18n.AUTO_REFRESH_ACTIVE}{' '}
+                  <EuiIconTip
+                    color="subdued"
+                    content={
+                      <FormattedMessage
+                        id="xpack.siem.footer.autoRefreshActiveTooltip"
+                        defaultMessage="While auto-refresh is enabled, timeline will show you the latest {numberOfItems} events that match your query."
+                        values={{
+                          numberOfItems: itemsCount,
+                        }}
+                      />
+                    }
+                    type="iInCircle"
+                  />
+                </b>
+              </EuiText>
+            ) : (
+              <PagingControl
+                data-test-subj="paging-control"
+                hasNextPage={hasNextPage}
+                isLoading={isLoading}
+                loadMore={loadMore}
+              />
+            )}
+          </EuiFlexItem>
+
+          <EuiFlexItem data-test-subj="last-updated-container" grow={false}>
+            <FixedWidthLastUpdated data-test-subj="fixed-width-last-updated" compact={compact}>
+              <LastUpdatedAt updatedAt={updatedAt || getUpdatedAt()} compact={compact} />
+            </FixedWidthLastUpdated>
+          </EuiFlexItem>
+        </FooterFlexGroup>
+      </FooterContainer>
+    </>
+  );
+};
+// TODO: check if it's necessary
```

Comment:
Did you mean to leave this TODO in there?

Reply:
nice catch, Thank you @watson 👍